### PR TITLE
Add python-dotenv recipe

### PR DESCRIPTION
**Pull request by proinsias**
xref: theskumar/python-dotenv#33

### CODE REVIEW

**Comment by proinsias** [comment]
@jakirkham – I think this is now RTM.

File: recipes/python-dotenv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - click >=5.0
+
+test:
+  imports:
+    - dotenv
+
+about:
+  home: https://github.com/theskumar/python-dotenv/
+  license: BSD 3-Clause
+#  license_file: LICENSE  # When added to MANIFEST.in – theskumar/python-dotenv#33
```
Comment (jakirkham) [inline recipes/python-dotenv/meta.yaml]: xref: https://github.com/theskumar/python-dotenv/pull/33

File: recipes/python-dotenv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - dotenv = dotenv:cli.cli
+requirements:
```
Comment (jakirkham) [inline recipes/python-dotenv/meta.yaml]: Maybe add a newline between these sections.

File: recipes/python-dotenv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+    - click >=5.0
+
+test:
+  imports:
+    - dotenv
+
+about:
+  home: https://github.com/theskumar/python-dotenv/
+  license: BSD 3-Clause
```
Comment (jakirkham) [inline recipes/python-dotenv/meta.yaml]: Could optionally add `license_family: BSD`.

**Comment by jakirkham** [comment]
Simple things to fix.

File: recipes/python-dotenv/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/python-dotenv/meta.yaml]: Done below.

**Comment by jakirkham** [comment]
Thanks @proinsias .

**Pull request merged**